### PR TITLE
Release 1.0.4

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

This PR merges the 1.0.4 release into stable/1.0

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: clients/go/internal/embedded/data/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.0.4-SNAPSHOT
+1.0.5
```
Comment (korthout) [inline clients/go/internal/embedded/data/VERSION:1]: ❓ Is this correct?

**Review by korthout — APPROVED** [review]
Looks good, but I'm not sure about a go thing

**Comment by korthout** [comment]
bors p=1

File: clients/go/internal/embedded/data/VERSION (hunk 1 above)
Comment (npepinpe) [reply clients/go/internal/embedded/data/VERSION:1]: Nope! Does this means I also added 1.0.5 as the next development version for the Java modules as well? :see_no_evil:

Comment (npepinpe) [reply clients/go/internal/embedded/data/VERSION:1]: Actually not - the poms show 1.0.5-SNAPSHOT, it's strange that the Go one removed the SNAPSHOT tag...Good catch. I'll look into it :thinking:

**Comment by npepinpe** [comment]
bors merge

What's bors p=1 ?

**Comment by korthout** [comment]
> What's bors p=1 ?

It's setting the priority for bors. It means that when you try to merge this, it will give it priority over other pulls. 

The things with this PR is that it changes the version, and anything we merge in the meantime kind of has the wrong versions in the pom and such. So I thought it made sense to give this a bit more prio. You could of course have merged other pulls to this same target in the meantime, but still I thought the extra prio might be nice.

**Pull request merged**